src/file-events/headers/generic_fsnotifier.h

```diff
@@ -59,8 +60,8 @@ class AbstractServer;
 
 class Command {
 public:
-    Command() {};
-    virtual ~Command() {};
+    Command(){};
```

Comment:
I don't understand what is happening here. When I saved the file, auto-format added the spaces and now your auto-format removes them again?

Reply:
Yeah, I saw this, too. Do you have a different formatter plugin maybe? I can't find which one does the thing. I have these enabled:

![image](https://user-images.githubusercontent.com/495366/77187453-cccfc180-6ad4-11ea-92dd-7b8350ec1e0f.png)
![image](https://user-images.githubusercontent.com/495366/77187512-e4a74580-6ad4-11ea-96fd-391221ca7b81.png)

Reply:
I only have those extensions: 
![image](https://user-images.githubusercontent.com/423186/77187836-60a18d80-6ad5-11ea-8cd3-750a6ea7a9e0.png)

Maybe the difference is that I am using a newer version of the `C/C++` extension (`0.27.0-insiders3`?

Reply:
That's one explanation. Another is that the formatter behaves differently on different OSs maybe?

Reply:
Yeah, I can reproduce your formatting with the `0.27-insiders3` version. Let's keep that version.